lib/Convos/Controller/Connection.pm

```diff
@@ -20,7 +20,13 @@ sub create {
   }
 
   eval {
-    $connection = $user->connection({protocol => $url->scheme, name => $name});
+    $connection = $user->connection(
+      {
+        name                => $name,
+        on_connect_commands => $json->{on_connect_commands} || [],
```

Comment:
I'm afraid that the IRC server will get confused of weird commands. We could however sanities the on_connect_commands in Irc.pm before sending them to the IRC server. Maybe that makes more sense..?